airbyte-server/src/test/java/io/airbyte/server/apis/BaseControllerTest.java

```diff
@@ -0,0 +1,260 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.server.apis;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import io.airbyte.analytics.TrackingClient;
+import io.airbyte.commons.server.handlers.*;
+import io.airbyte.commons.server.scheduler.SynchronousSchedulerClient;
+import io.airbyte.commons.temporal.TemporalClient;
+import io.airbyte.db.Database;
+import io.micronaut.context.annotation.Replaces;
+import io.micronaut.context.annotation.Requires;
+import io.micronaut.core.util.StringUtils;
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.HttpStatus;
+import io.micronaut.http.client.HttpClient;
+import io.micronaut.http.client.annotation.Client;
+import io.micronaut.http.client.exceptions.HttpClientResponseException;
+import io.micronaut.runtime.server.EmbeddedServer;
+import io.micronaut.test.annotation.MockBean;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import io.temporal.client.WorkflowClient;
+import io.temporal.serviceclient.WorkflowServiceStubs;
+import jakarta.inject.Inject;
+import jakarta.inject.Named;
+import javax.sql.DataSource;
+import lombok.extern.slf4j.Slf4j;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.Condition;
+import org.assertj.core.api.InstanceOfAssertFactory;
+import org.jooq.DSLContext;
+import org.mockito.Mockito;
+
+@Slf4j
+@MicronautTest
+@Requires(property = "mockito.test.enabled",
+          defaultValue = StringUtils.TRUE,
+          value = StringUtils.TRUE)
+@SuppressWarnings("PMD.AbstractClassWithoutAbstractMethod")
+public abstract class BaseControllerTest {
+
+  AttemptHandler attemptHandler = Mockito.mock(AttemptHandler.class);
```

Comment:
Did you mean to make everything package scoped in this class?

Reply:
Yes, It is a recommendation of pmd. don't think that the base will be use outside of the package. I need to remove the public scope.